src/task30/task30.js

```diff
@@ -0,0 +1,34 @@
+function insideTheBox() {
+    let cx = document.querySelector("canvas").getContext("2d");
```

Comment:
Move out as much variables as you find reasonable to be global.

Reply:
but that's the point! I have wrapped the code in to the function, because idk how to make html file see the js otherwise